apps/email_management/apps.py

```diff
@@ -0,0 +1,55 @@
+from django.apps import AppConfig
+import logging
+from djangocms_forms.signals import form_submission
+from django.core.mail import send_mail
+
+logger = logging.getLogger(f"portal.{__name__}")
+
+def send_confirmation_email(form_name, form_data):
+
+    from django.contrib.sites.models import Site
+
+    # Get the current site
+    current_site = Site.objects.get_current()
+
+    # Get the site name
+    site_name = current_site.name
+
+    text_body = f"""
+            Greetings,
+            
+            You have successfully submitted a form on the {site_name} website. Thank you for your submission.
+
+            Sincerely,
+            {site_name} Communications
+    """
+    email_body = f"""
+            <p>Greetings,</p>
+            <p>
+                You have successfully submitted a form on the {site_name} website. Thank you for your submission.
+            </p>
+            <p>
+            Sincerely,<br>
+            {site_name} Communications
+            </p>
```

Comment:
**Request.** Change indentation. Ideas:

- Do not indent.
- Teach me the reasoning behind 8 spaces.
- Indent with four spaces, like the rest of the file.

Reply:
Fixed.